src/lockfile/wrapper.js

```diff
@@ -86,9 +87,12 @@ export default class Lockfile {
     [key: string]: LockManifest
   };
 
-  static async fromDirectory(dir: string, reporter?: Reporter): Promise<Lockfile> {
+  static async fromDirectory(config: Config, dir: string, reporter?: Reporter): Promise<Lockfile> {
```

Comment:
nit: this couples lockfile wrapper to config, not sure if it is a concern but maybe we'll want to decouple it back in the future